spec/regression/GH-7234_math_ldexp_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe "Math.ldexp" do
+
+  it "returns correct value that closes to the max value of double type" do
```

Comment:
could you move these tests to https://github.com/ruby/spec/blob/master/core/math/ldexp_spec.rb ? truffleruby has the same issue

Reply:
I will do it tomorrow.

Reply:
@ahorek 
See https://github.com/ruby/spec/pull/928 .
Is this okay?

Reply:
great, thanks!